src/views/Settings.vue

```diff
@@ -45,18 +45,64 @@
         <v-card flat tile>
           <v-card-title v-text="'Backend'" />
           <v-card-text>
-            <v-text-field
-              label="NEST Server"
-              v-model="state.app.nestServer.url"
-            />
-            <span v-if="state.nestVersion">
-              Response:
-              <v-chip color="orange darken-3" dark small>
-                <v-avatar left>
-                  <v-icon small v-text="'mdi-checkbox-marked-circle'" />
-                </v-avatar>
-                NEST version: {{ state.nestVersion }}
-              </v-chip>
+            <v-tooltip top open-delay="300">
+              <template v-slot:activator="{ on, attrs }">
+                <v-text-field
+                  label="NEST Server URL"
+                  placeholder="https://www.my-nest-server.com:5000"
+                  v-model="state.app.nestServer.url"
+                  v-bind="attrs"
+                  v-on="on"
+                />
+              </template>
+              <span
+                >Please enter the URL where the NEST Server can be found at
+                (including protocol!).</span
+              >
```

Comment:
```suggestion
              <span>
                Please enter the URL where the NEST Server can be found at
                (including protocol!).
              </span>
```

Reply:
Ok, I adressed that issue in the commit below.